tests/integration/conftest.py

```diff
@@ -25,6 +26,11 @@ def app_config(app_config):
     # add extra global config if you would like to customize the config
     # for a specific test you can chagne create fixture per-directory
     # using ``conftest.py`` or per-file.
+    app_config[
+        "DEBUG"
+    ] = (
+        False
```

Comment:
Do we have specific explanation why this is failing?

Reply:
No, we don't . It's a known issue, and it was known when we were implementing alembic in inspire-next. Nothing changed from this time.  This may be similar problem to this https://chase-seibert.github.io/blog/2015/06/12/flask-werkzeug-reloader-python-dash-m.html# but I never had time ti investigate it.